sweep/sweeper.go

```diff
@@ -1513,6 +1513,17 @@ func (s *UtxoSweeper) updateSweeperInputs() InputsMap {
 			continue
 		}
 
+		// If the input has a deadline that's two weeks away, we won't
+		// attempt to create a sweeping tx and fee bump it now.
+		deadline := input.params.DeadlineHeight.UnwrapOr(0)
+		if deadline > s.currentHeight+DefaultDeadlineDelta*2 {
```

Comment:
I think we should rather not sweep the anchor at all then try to do this, because we will attempt to sweep our anchor here way passed 16 blocks have passed and a third party can easily sweep it. And there are sweepers out there which do this, when we now try to collect the anchor after 8 weeks, we might really run into the issue of bumping bing anchor-collector transactions out of the mempool, which can stay there forever if the fees are higher than ~6 sats/vbyte.

My recommendation is to not sweep them at all, because waiting 8 weeks before we attempt to sweep them is almost equal to not sweeping them but bears the risk of running into some RBF races against those big AnchorCollector transactions.